report.html

```diff
@@ -1,93 +1,4 @@
-            <ul>
-                <li>
-                    <a href="index.html">Home Page</a>
-                </li>
-                <li>
-                    <a href="call-for-participation.html">Call for Participation</a>
-                </li>
-                <li>
-                    <a href="agenda.html">Workshop Agenda</a>
-                </li>
-                <li>
-                    <a href="minutes.html">Minutes</a>
-                </li>
-                <li>
-                    <a href="position-statements.html">Position Statements</a>
-                </li>
-                <li>
-                    <a  class="disabled">Report</a>
-                </li>
-            </ul>
-        </nav>
-    </header>    
-    <aside class="box box-host" id="host">
-        <h2 class="footnote">Host</h2>
-        <p class="image-paragraph-host">
-            W3C gratefully acknowledges Natural Resources Canada for hosting this workshop.
-        </p>
-        <p class="image-paragraph">
-            <a href="https://www.nrcan.gc.ca/earth-sciences/geomatics/canadas-spatial-data-infrastructure/10783" class="image-link">
-            <img src="assets/media/your-logo-here.png" alt="Your logo here"></a>
-        </p>
-    </aside>
-    <aside class="box" id="sponsor">
-        <h2 class="footnote">Sponsors</h2>
-            <p>
-                Platinum, Gold and Silver Sponsorships available
-            </p>
-        <section class="sponsors-section ssponsors-section-platinum">
-            <h3 id="platinum-sidebar">Platinum Sponsors</h3>
-            <p>
-                <a href="" class="image-link"><img src="" alt=""></a>
-            </p>
-        </section>
-        <section class="sponsors-section sponsors-section-gold">
-            <h3 id="gold-sidebar">Gold Sponsors</h3>
-            <p>
-                <a href="" class="image-link"><img src="" alt=""></a>
-            </p>
-        </section>
-        <section class="sponsors-section sponsors-section-silver">
-            <h3 id="silver-sidebar">Silver Sponsors</h3>
-            <p>
-                <a href="" class="image-link"><img src="" alt=""></a>
-            </p>
-            
-            <h3 id="supporters-sidebar">Supporters</h3>
-        <p class="image-paragraph">
-          <a href="" class="image-link"><img src="" alt=""></a>	  
-            </p>
-        </section>
-        <p>
-            <a href="sponsorship.html">Sponsorship package info.</a>
-        </p>
-    </aside>        
-    <h1>Workshop Report</h1>
-    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
Thanks for clearing this out!

Looks like the lorem ipsum was (accidentally?) pushed live in #3 when @prushforth was adding placeholder text & playing around with trying to get the content longer than the sidebars.

PS, @nchan0154, if you have some more time later in the week, maybe see if you can refactor the layout to modern CSS so that it still looks good even while we don't have much content in the main content.

Reply:
No, I pushed it intentionally.  Perhaps I cared too little in this case.

Reply:
I don't know if it matters, but I set the github.io site to use the master branch; I notice this wants to merge into the gh-pages branch?

Reply:
>  I notice this wants to merge into the gh-pages branch

Good check, but it's the opposite; it's merging _from_ a gh-pages branch on Nic's fork.

Reply:
I added a clearfix so that the boxes don't break out of the layout if shorter than the main content, but I'm certainly up for refactoring to a more modern solution than floats 😂. I think there is some flexibility as to which branch this is, but I've never managed a bunch of github pages before.